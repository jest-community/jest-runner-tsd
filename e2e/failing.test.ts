import { expect, test } from '@jest/globals';
import runJest from './runJest';

test('works with failing test', async () => {
  expect(await runJest('failing')).toMatchSnapshot();
});
