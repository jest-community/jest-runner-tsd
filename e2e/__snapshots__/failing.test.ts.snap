// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`works with failing test 1`] = `
"FAIL e2e/__fixtures__/failing/index.test.ts
  âœ“
e2e/__fixtures__/failing/index.test.ts:5:19 - error - Argument of type 'number' is not assignable to parameter of type 'string'.
Test Suites: 1 failed, 1 total
Tests:       1 failed, 1 passed, 2 total
Snapshots:   0 total
Time:
Ran all test suites.
"
`;
